### DISCUSSION TITLE
Lower visual quality for sprites when switching from bevy 0.9.0 to 0.9.1

### DISCUSSION THREAD

**Discussion by GuillaumeGomez** [bevyengine/bevy#6950] [General]
Hi,

I'm not sure if it's a bug so I opened a discussion instead. Switching to bevy 0.9.1 greatly reduced the visual quality of the sprintes and I now have this rendering:

![image](https://user-images.githubusercontent.com/3050060/207598657-014781dd-8269-49f1-9f3f-392fdf1c6092.png)

To be compared with 0.9.0 (for the background, I can change it with `ClearColor` I suppose):

![fe9144826e6034b4](https://user-images.githubusercontent.com/3050060/207602530-897ef1af-f637-4bc5-b819-34865400bdef.jpeg)

If anyone has an idea what might have changed, it'd be very helpful. Here is the repository: [github.com/GuillaumeGomez/rust-toy-game](https://github.com/GuillaumeGomez/rust-toy-game).

**Comment by pkupper** [comment]
Are you talking about the light gray pixels around the sprites? It looks like they are part of the texture and just happen to match the old `ClearColor`. Therefore they were not visible on the previous background but are on the darker background.

**Reply by GuillaumeGomez** [reply]
Yes I am and the sprites don't have such a "border":

![image](https://user-images.githubusercontent.com/3050060/207606066-516a0128-d110-4c77-bf8b-3441223c35be.png)

You can see pretty that there is no such border around the ears.

**Reply by pkupper** [reply]
Then it might be a blending issue. The gray pixels might have an alpha value close to but not exacly zero and blending seems to be disabled. Possibly related to: https://github.com/bevyengine/bevy/issues/6315

**Reply by GuillaumeGomez** [reply]
Interesting. I didn't know that depending on what settings you have in your image, it might impact the rendering like this. It seems to be a bug indeed. But I'm not sure why it worked before though...

**Reply by pkupper** [reply]
For 3D, the `StandardMaterial` type has a field `alpha_mode` to specify how the alpha value is used (Blending or Mask). But I don't see a way to configure this for ColorMaterial in the docs. I might be missing something.

**Reply by GuillaumeGomez** [reply]
I just realized that I was using the Intel Integrated GPU and not my NVIDIA one so I just updated the driver and switched back to using the NVIDIA driver and now it creates a window but seems to be stuck in an infinite loop. Going to open an issue.

**Reply by GuillaumeGomez** [reply]
Opened the issue: https://github.com/bevyengine/bevy/issues/6951
